### ISSUE TITLE
Rename child monitors

### ISSUE THREAD

**Issue by helfi92** [taskcluster/taskcluster#2601]
From https://github.com/taskcluster/taskcluster/pull/2600#discussion_r403182785:

> hmm, I think this should be notify.childMonitor('db'). I think I r+'d a patch where we did this with service name in the other services. could we update them all to be like this in this PR?

**Comment by helfi92** [comment]
@imbstack Do you remember where that was? I wasn't able to to find what you were referring to with `git grep childMonitor`.

**Comment by djmitche** [comment]
For example, `services/queue/src/main.js`
```
  pulseClient: {
    requires: ['cfg', 'monitor'],
    setup: ({cfg, monitor}) => {
      return new pulse.Client({
        namespace: 'taskcluster-queue',
        monitor: monitor.childMonitor('pulse-client'),                                                   
        credentials: pulse.pulseCredentials(cfg.pulse),
      }); 
    },  
  },  
```

I'll fix this up in #2522.
